ocaml/xapi/nm.ml

```diff
@@ -364,6 +364,8 @@ let bring_pif_up ~__context ?(management_interface=false) (pif: API.ref_PIF) =
           let cleanup, bridge_config, interface_config = create_bridges ~__context rc net_rc in
           List.iter (fun (name, force) -> Net.Bridge.destroy dbg ~name ~force ()) cleanup;
           Net.Bridge.make_config dbg ~config:bridge_config ();
+          let igmp_snooping = Db.Pool.get_igmp_snooping_enabled ~__context ~self:(Helpers.get_pool ~__context) in
```

Comment:
I prefer to do this at the end of the function, near where the MTU field is updated in a similar way. And ignore but log exceptions, because this is not business critical. Also, do the set only is the field has changed. All the same pattern as below under `(* sync MTU *)`.